fml/platform/posix/native_library_posix.cc

```diff
@@ -13,8 +13,8 @@ NativeLibrary::NativeLibrary(const char* path) {
   ::dlerror();
   handle_ = ::dlopen(path, RTLD_NOW);
   if (handle_ == nullptr) {
-    FML_DLOG(ERROR) << "Could not open library '" << path << "' due to error '"
-                    << ::dlerror() << "'.";
+    FML_LOG(ERROR) << "Could not open library '" << path << "' due to error '"
```

Comment:
Since this will add a little misleading log spam on systems where falling back to the fully qualified path is expected, let's file an issue and add a TODO() here to shift this back to a `DLOG` when the investigation is complete.